src/io/github/humbleui/ui.clj

```diff
@@ -561,15 +561,19 @@
       (when (= :mouse-move (:event event))
         (let [hovered?' (.contains ^IRect child-rect (IPoint. (:x event) (:y event)))]
           (when (not= hovered? hovered?')
+            (when on-hover (on-hover))
```

Comment:
Shouldn’t this be either `(when (and on-hover hovered?')` or `(on-hover hovered?')`? Because there are two events, one the mouse gets over and one when it moves away

Reply:
Yes you're right, it should be one of those. Either works for me, which do you prefer?

Reply:
How about `{:keys [on-hover on-out] :as opts}` as first parameter? One optional callback for when hovered and one for when out. `(on-hover false)` feels a bit counter-intuitive

Reply:
Good idea. I made the change and threw in a quick docstring, too.